File: Level1/2016년.py
def solution(a, b):
    max_day = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    days = ["THU","FRI","SAT","SUN","MON","TUE","WED"]
    return days[(sum(max_day[:a-1])+b)%7]

File: Level1/2016년.js
function solution(a, b) {
    let max_day = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
    let days = ["THU","FRI","SAT","SUN","MON","TUE","WED"]
    if (a === 1)
        return days[b % 7]
    else {
        return days[(max_day.slice(0, a - 1).reduce((x, y) => x + y) + b) % 7]
    }
}
